lib/ex_aws/sqs/parsers.ex

```diff
@@ -175,6 +179,55 @@ if Code.ensure_loaded?(SweetXml) do
       {:ok, Map.put(resp, :body, parsed_body)}
     end
 
+    def message_attributes_to_map(list_of_message_attribtues) do
+      list_of_message_attribtues
+      |> Enum.reduce(%{}, fn(%{name: name, data_type: data_type} = attr, acc) ->
+        case parse_attribute_value(data_type, attr) do
+          ^attr ->
+            Map.put(acc, name, attr)
```

Comment:
Probably a dumb question, but why do we not want a value and data type map here?